pkg/brokerapi/schemas.go

```diff
@@ -16,16 +16,16 @@ limitations under the License.
 
 package brokerapi
 
-// Schemas represents a broker's schemas for both service instances and service
-// bindings
+// Schemas represents a plan's schemas for service instance and binding create
+// and update.
 type Schemas struct {
-	Instance Schema `json:"instance"`
-	Binding  Schema `json:"binding"`
+	ServiceInstances *Schema `json:"service_instances,omitempty"`
+	ServiceBindings  *Schema `json:"service_bindings,omitempty"`
```

Comment:
It changes the application-layer messages, but they were not correct in terms of the actual proposed change to begin with.  They are correct after this PR.